### PR TITLE
feat(server): Removed OvernightJS

### LINKED ISSUE #231: Remove OvernightJS from the server
The NodeJS server implementation currently uses OvernightJS as a wrapper around Express for building a more expressive API. While building the API we started to work around OvernightJS because it's too limited for our use. Therefore we want to remove OvernightJS as dependency and use plain Express.

### PR DESCRIPTION
**Pull request by BasMasking**
Fixes #231 

Changes proposed in this pull request:
- removed the overnightJS dependency

@MaskingTechnology/jitar

### CODE REVIEW

File: packages/server-nodejs/src/JitarServer.ts [hunk 1]
```diff
@@ -36,20 +35,25 @@ const STARTUP_MESSAGE = `
   By Masking Technology (masking.tech)
 `;
 
-export default class JitarServer extends OvernightServer
+export default class JitarServer
 {
+    #app: Express;
+    //#router: express.Router;
```
Comment (petermasking) [inline packages/server-nodejs/src/JitarServer.ts:41]: I don't think the router will come back...

**Review by petermasking — CHANGES_REQUESTED** [review]
I think we can also remove the "experimentalDecorators" from the tsconfig file.

**Comment by BasMasking** [comment]
> I think we can also remove the "experimentalDecorators" from the tsconfig file.

We still have the decorators for the runtime configurations. If we replace these validations by, for example, zod, we can remove the decorators property.

File: packages/server-nodejs/src/JitarServer.ts (hunk 1 above)
Comment (BasMasking) [reply packages/server-nodejs/src/JitarServer.ts:41]: Done

**Review by petermasking — APPROVED** [review]

**Pull request merged**